source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -124,7 +124,9 @@ def _getPointFromOffset(self,offset):
 			res=self.obj.IAccessibleTextObject.characterExtents(offset,IAccessibleHandler.IA2_COORDTYPE_SCREEN_RELATIVE)
 		except COMError:
 			raise NotImplementedError
-		point=textInfos.Point(res[0]+(res[2]/2),res[1]+(res[3]/2))
+		# Using the center of the character's bounding rectangle could result in a point
+		# of which IA2 believes that it belongs to a higher offset.

```

Comment:
1. Did you see this bug in Firefox? IMO, this is an implementation bug; the centre of a character should be treated as part of that character. I guess this should be safe enough for characters, though. It can get nasty with objects due to borders, etc., which is why we tend to use the centre. Anyway, it might be worth noting in the comment that you saw this bug occur in Firefox.
2. How does this bug actually manifest from a user perspective? I assume clicking does click the right character, but maybe mouse tracking reports the wrong one?
3. It's not clear why you're talking about the centre here unless you're reading this diff. Perhaps you could preface it with "Normally, we tend to use the centre point of a bounding rectangle."